### PR TITLE
SC-200: Add community mgr access to billing dashboard

### PR DESCRIPTION
**Pull request by xschildw**
This PR adds access to the Billing dashboard through a JumpCloud application.
- should it be split from jd-idp?
- the permission is too broad, looking to restrict to cost explorer only

### CODE REVIEW

File: templates/develop/jumpcloud-idp.yaml [hunk 1]
```diff
@@ -31,6 +34,43 @@ Resources:
...
+  AWSIAMCommunityMgrAccessPolicy:
+    Type: "AWS::IAM::ManagedPolicy"
+    Properties:
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          -
+            Effect: "Allow"
+            Action:
+              - aws-portal:ViewBilling
```
Comment (zaro0508) [inline templates/develop/jumpcloud-idp.yaml:71]: is this the same as the built in `BillingViewAccess`?

Comment (xschildw) [reply templates/develop/jumpcloud-idp.yaml:71]: Haven't seen that, there was a Billing job function policy (which we use for admins) but that's even more lax. I'll look it up.

**Comment by zaro0508** [comment]
any updates on this @xschildw ?

**Comment by xschildw** [comment]
@zaro0508 , last thing I did was check https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/billing-permissions-ref.html, and add perms to modify reports. Just pushed that up.
If limited number of SageBio employees have access then it's fine.
We should have processes generating reports for external consumption, correct?
If OK, I'd like to merge and see first-hand what experience is.

**Pull request merged**